sig-guidance.md

```diff
@@ -0,0 +1,23 @@
+# SIG Guidance
+
+A collection of guidance for Special Interest Groups.
+
+## Meeting Notes
+
+Notes must be taken during SIG meetings and made available to the public, which are linked from the SIG's README.md file in the governance repository.
+
+To maintain good cadence, it is generally recommended to have a scribe taking notes who is not the person facilitating the meeting or presenting.
+
+## Meeting Recordings
+
+All SIG meetings should be recorded. 
```

Comment:
Is this a "should" or a "must"? I.e. would it be acceptable for a SIG that has members that are uncomfortable with recordings to decide that they will _not_ record their meetings?

Reply:
I think that it is important that notes be taken and publicized.  I disagree with the position that all meetings *must* be recorded, as I think it may discourage participation.  Watching a recording is fine if you are interested in consuming information.  It does not work if you want to provide input.

Reply:
Right now, I'm positioning it as a "should" - optional, but encouraged. 

A lot of the community is asking for recordings because they cannot participate due to timezones or other reasons. With this feedback, I'd like to move to mandatory recordings eventually, but would like to make that decision more deliberately once we've established whether or not it does indeed hinder participation, and if the community is getting use out of it.

Reply:
Could we emphasize that "should" means "optional", unless we think that'll be clear to readers?

Thanks for the clarification!

Reply:
How about the following:

We encourage SIGs to record their meetings so they are available to people in other timezones.  At the same time, we understand that recording meetings may hinder participation in the meetings, so SIGs may choose not to record meetings in general or specific instances.  If a SIG chooses to not record a meeting, please inform the SC and TOC so that we have awareness of how often this is happening.  It is particularly important that notes are taken and published for any meeting that is not recorded.

Reply:
This sounds good, I'll update with this exact language.